go/service/assets/api_op_DeleteAsset.go

```diff
@@ -2,6 +2,7 @@ package assets
 
 import (
 	"fmt"
+	"io/ioutil"
```

Comment:
As suggested by @Nazdroth, maybe we can use the `io` package instead of the `io/ioutil`. I think it becomes a standard now. wdyt?

Reply:
my bad 😅  fixed now! 😄